src/openforms/submissions/form_logic.py

```diff
@@ -36,13 +38,29 @@ def evaluate_form_logic(
     step: "SubmissionStep",
     data: Dict[str, Any],
     dirty=False,
+    **context,
```

Comment:
Is this (`**context`) broader than just "request" for a reason? Seems like request is the only thing that gets passed.

If `**context` is usefull, please document what it should/can contain. in this function.

Reply:
not really at this stage, though I figured I would just pass down the serializer context and it's up to the downstream functions to check if the context contains what it needs.